PD-442 missing license events and PD-446 describe the `scalesize` parameter (#728)
portworx/pxdocs: content/portworx-install-with-kubernetes/autopilot/reference/_index.md

* **scaletype**: Specifies the type of operation to be performed to expand the pool. Supported values are:
    * *add-disk*: Portworx will add new disk(s) to the existing storage pool
    * *resize-disk*: Portworx will resize existing disks in the storage pool

> * **scalesize**: Specifies the amount, in GiB or TiB, by which Autopilot should expand a storage pool.

Reviewer: this needs to be `Gi or Ti`

```suggestion
* **scalesize**: Specifies the amount, in Gi or Ti, by which Autopilot should expand a storage pool.
```

> * **scalesize**:  Specifies the amount, in GiB or TiB, by which Autopilot should exapnd a storage pool.

Reviewer: type in `exapnd`

      scaletype: "add-disk"
```

> Expand the pool by 100GiB by resizing disks

Reviewer: ```suggestion
Expand the pool by 100Gi by resizing disks
```

```text
  actions:
  - name: openstorage.io.action.storagepool/expand
    params:

>       scalesize: "100GiB"

Reviewer: Sorry, I think this needs to be `100Gi` (and not `100GiB`).

>       scalesize: "100GiB"

Author: ```suggestion
      scalesize: "100Gi"
```
